network/p2p/blob_service.go

```diff
@@ -84,9 +88,34 @@ func NewBlobService(
 
 	cm := component.NewComponentManagerBuilder().
 		AddWorker(func(ctx irrecoverable.SignalerContext, ready component.ReadyFunc) {
-			bs.blockService = blockservice.New(bs.blockStore, bitswap.New(ctx, bsNetwork, bs.blockStore, bs.config.BitswapOptions...))
+			btswp := bitswap.New(ctx, bsNetwork, bs.blockStore, bs.config.BitswapOptions...).(*bitswap.Bitswap)
+			bs.blockService = blockservice.New(bs.blockStore, btswp)
 
 			ready()
+
+			ticker := time.NewTicker(15 * time.Second)
+			for {
+				select {
+				case <-ctx.Done():
+					return
+				case <-ticker.C:
+					stat, err := btswp.Stat()
+					if err != nil {
+						logger.Err(err).Str("component", "blob_service").Str("prefix", prefix).Msg("failed to get bitswap stats")
```

Comment:
Does this need to be
```suggestion
						logger.Error().Err(err).Str("component", "blob_service").Str("prefix", prefix).Msg("failed to get bitswap stats")
```

Reply:
nevermind. looks like `Err()` will implicitly add `Error()`